sources/packages/backend/apps/api/src/route-controllers/supporting-user/models/supporting-user.dto.ts

```diff
@@ -55,10 +71,23 @@ export class UpdateSupportingUserAPIInDTO {
   @IsNotEmpty()
   sin: string;
   @IsNotEmpty()
-  studentsDateOfBirth: string;
-  @IsNotEmpty()
   @MaxLength(USER_LAST_NAME_MAX_LENGTH)
   studentsLastName: string;
+  @IsEnum(SupportingUserType)
+  supportingUserType: SupportingUserType;
+  /**
+   * For Parent search only. Required for Parent, Optional for Partner.
+   */
+  @ValidateIf((o) => o.supportingUserType === SupportingUserType.Parent)
+  @IsNotEmpty()
+  @MaxLength(SUPPORTING_USER_FULL_NAME_MAX_LENGTH)
+  parentFullName?: string;
```

Comment:
https://github.com/bcgov/SIMS/pull/4872#discussion_r2193260067 same here.

Reply:
Thanks for updating the property. But validate if is not typed. 

![image](https://github.com/user-attachments/assets/9400f76a-77d6-43d6-87c9-ed3d9af389fe)